core/domain/stats_domain.py

```diff
@@ -249,26 +267,57 @@ def __init__(
 
     @property
     def total_answers_count(self):
+        """Returns the total number of answers submitted to this state.
+
+        Returns:
+            int. The total number of answers submitted to this state.
+        """
         return self.total_answers_count_v1 + self.total_answers_count_v2
 
     @property
     def useful_feedback_count(self):
+        """Returns the total number of answers that received useful feedback.
+
+        Returns:
+            int. The total number of answers that received useful feedback.
+        """
         return self.useful_feedback_count_v1 + self.useful_feedback_count_v2
 
     @property
     def total_hit_count(self):
+        """Returns the total number of times the state was entered.
+
+        Returns:
+            int. The total number of times the state was entered.
+        """
         return self.total_hit_count_v1 + self.total_hit_count_v2
 
     @property
     def first_hit_count(self):
+        """Returns the number of times the state was entered for the first time.
+
+        Returns:
+            int. The number of times the state was entered for the first time.
+        """
         return self.first_hit_count_v1 + self.first_hit_count_v2
 
     @property
     def num_completions(self):
+        """Returns total number of times the state was completed.
+
+        Returns:
+            int. The total number of times the state was completed.
+        """
         return self.num_completions_v1 + self.num_completions_v2
 
     @property
     def num_times_solution_viewed(self):
+        """Returns the number of times the solution button was triggered.
+
+        Returns:
+            int. Number of times the solution button was triggered to answer a
+                state (only for version 2).
```

Comment:
Let's be clearer: `only for events with schema version 2`

Reply:
Done!